google/cloud/storage/README.md

```diff
@@ -44,7 +44,14 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
 
 ## Release Notes
 
-### v1.8.x - TBD
+### v1.9.x - TBD
+
+### v1.8.x - 2020-01
+
+* feat: implement non-resumable parallel uploads (#3279)
```

Comment:
I think we should advertise a feature that improves performance from ~50MiB/s to about 1300MiB/s a bit more strongly...  #3279  Maybe:

* feat: implement parallel uploads. For large files this can improve the upload bandwidth by a factor of 20. Note that in this release such uploads cannot be resumed after a program restart.

Reply:
Done.